src/components/common/list-item/components/list-item-status/list-item-status.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { VFC } from 'react';
+import { View } from 'react-native';
+
+import { ButtonIcon } from '~components/common/button-icon';
+import globalStyles from '~global/constants.style';
+import styles from './list-item-status.style';
+
+export const ListItemStatus: VFC<Props> = ({
+  isTodoSolelyEditing,
+  isTodoSelectedOrCompleted,
+  isMultipleEditingMode,
+  onSelectTodoHandler,
+  onCheckPressTodoHandle,
+}) => {
+  return (
+    <View style={styles.complete}>
+      {isTodoSolelyEditing ? (
+        <ButtonIcon testID="pencil" variant="pencil" size={globalStyles.ICON_EXSM_SIZE} />
+      ) : isTodoSelectedOrCompleted ? (
```

Comment:
Might be better to move to a separate function that return component

Reply:
Moved 'else' part to separate function